source/agg-pipeline-builder/export-pipeline-to-language.txt

```diff
@@ -6,18 +6,74 @@ Export Pipeline to Specific Language
 
 .. default-domain:: mongodb
 
-*New in version 1.15.0*
-
 You can use the :ref:`Aggregation Pipeline Builder
-<compass-agg-builder>` to export your finished pipeline to one of the
-supported languages; Java, Node, C#, and Python 3. This feature
-facilitates formatting and exporting pipelines created in |compass| for
-use in your applications.
+<compass-agg-builder>` to export your finished pipeline to a language of your 
+choice to use in your application. |compass-short| facilitates the formatting 
+and exporting of pipelines created with Aggregation Pipeline Builder.
+
+About this Task 
+---------------
+
+You can export your pipeline to the following languages: 
+
+- Java
```

Comment:
[question]

Should we alphabetize these?

Reply:
Will do! Originally kept it in the order it currently is because that's how it's shown in the Compass UI.